### PR TITLE
Merge Adorable master to LMHT master (deploy)

### PR DESCRIPTION
**Pull request by ericanaughton**
:tea: 

Ready to go

This PR contains that have already been approved individually, just need a final check before deploying to the live site.
- Fixes link to CONTRIBUTING.md files in README
- Changes contributing.md to CONTRIBUTING.md
- Summarizes and shortens CONTRIBUTING.md
- Makes the Welcome Section the default page
- Adds links to items within the Welcome Section
- Adds a ribbon with a link to the CONTRIBUTING.md
- Makes style changes to ribbon
- Alphabetizes and correctly orders styles and classes within CSS
- Updates the design of audio player
- Adds voice English and Spanish voice recordings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+1.  Fork the repository and clone it locally. Attach your local to the upstream repository by adding it as a remote. Pull the changes from upstream so that you are modifying current data when you submit a pull request. More in-depth instructions [here].
+2.  Create a branch for your edits. Depending on what you have loaded in your editor, `git checkout -b MyBranch` should suffice.
+3.  It is common practice to name your branch according to what it is fixing or updating. Before submitting these changes, verify that they work.
+4.  Because the project uses Ajax, you must run a local web server to see your changes.
+
+```
+$  cd myrepo
+$  python -m SimpleHTTPServer
+*  Running on [http://0.0.0.0:8000](http://0.0.0.0:8000/)
```
Comment (werbeckes) [inline CONTRIBUTING.md]: This is in a code block so the typical markdown pieces don't work. (tldr: this doesn't make a link)

**Comment by werbeckes** [comment]
:+1: with minor change to code block to remove link

File: elements/a.html [hunk 2]
```diff
@@ -47,3 +47,7 @@
     </code>
   </pre>
 </article>
+
+
+
+
```
Comment (ericanaughton) [inline elements/a.html]: trailing whitespace

**Comment by ericanaughton** [comment]
:ship: :city_sunset: 

this comment is from brian at tenforward. he looked over it briefly, so i dont know if it actually counts as a thumb.

**Comment by jremsik-gpsw** [comment]
:+1:

**Comment by bigtiger** [comment]
:+1: Once adorableio/lmht#18 is merged

**Comment by ericanaughton** [comment]
is this good to go then @bigtiger ?

**Comment by bigtiger** [comment]
Yiss

**Pull request merged**